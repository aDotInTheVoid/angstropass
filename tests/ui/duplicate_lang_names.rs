#[nanopass::langs]
mod langs {
    mod foo {}
    mod foo {}
}

fn main() {}
