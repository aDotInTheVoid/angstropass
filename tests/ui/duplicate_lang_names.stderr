error: duplicate language: foo
 --> tests/ui/duplicate_lang_names.rs:4:9
  |
4 |     mod foo {}
  |         ^^^
